BulletConfigVersion.cmake.in

```diff
@@ -0,0 +1,18 @@
+#                                               -*- cmake -*-
+#
+#  BulletConfigVersion.cmake(.in)
```

Comment:
Is there any particular reason that you're not generating this with `WRITE_BASIC_PACKAGE_VERSION_FILE`? You've upgraded to CMake 3.0 anyways, so this is available to you.

Reply:
I introduce it later in the commit `Export Bullet::* CMake targets so users can simply link to them`

Reply:
Oh you're right, I'm using the `WRITE_BASIC_PACKAGE_VERSION_FILE` function but I forgot to remove `BulletConfigVersion.cmake.in`

Reply:
Force pushed a fix.